openj9.build/makefile

```diff
@@ -447,6 +455,7 @@ test: $(TEST_TARGETS)
 test.daa: $(DAA_TESTS)
 test.gc: $(GC_TESTS)
 test.SharedClasses: $(SHARED_CLASSES_TESTS)
+test.JlmTests: $(JLM_TESTS)
```

Comment:
Maybe a bit picky, but I don't think we should have 'Tests' in the end of the target (we haven't got it for the other targets and the target starts with test. anyway).  So test.jlm or test.Jlm (all lower case is easier to type)?

Reply:
I agree, but it's going to be quite a large change if we want to change the name of the STF test plugin class names now. We have to update 1) class names 2) playlist entries 3) test names in the make file. And we have to do it for both openjdk.test.jlm project (that too has test class names starting with Test (e.g. TestJlmLocal.java, etc)) and openj9.test.jlm project. I think if we want to do this refactoring of names, we should do it in a separate PR.

Reply:
I only meant change the name of the make target (test.JlmTests  --> test.jlm  (= run all the jlm tests), not the names of the test plugins.

Reply:
Oh.. I misunderstood :)
I have updated the PR with the change. Please have a look.